ArmyOfOnes/ArmyOfOnes/ViewControllers/CurrenciesListViewController.swift

```diff
@@ -0,0 +1,216 @@
+//
+//  CurrenciesListViewController.swift
+//  ArmyOfOnes
+//
+//  Created by Fabian Mariño on 11/8/16.
+//  Copyright © 2016 Fabian Mariño. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+enum LayoutConstraints: CGFloat {
+    case inputOffset = 14,
+    topInset = 43,
+    tableTopInset = 70
+}
```

Comment:
When using `enum`s for "magic numbers", please do not define a type and use instead static variables. Also, I'd suggest to group all these variables into a single enum that could be called `CurrenciesListViewController`. That way you won't have naming conflicts with other config structs.
```
enum CurrenciesListViewControllerConfig {
   static let inputOffset       = CGFloat(14)
   static let topInset            = CGFloat(43)
   static let tableTopInset  = CGFloat(70)
}
```

Another way to avoid the struct having such long names is to define the struct inside the class/struct where it is going to be used and only call it `Config`.

```
class CurrenciesListViewController: UITableViewController {
   enum Config {
      ...
   }
   
   ...
}
```
This way inside the `CurrenciesListViewController` it can be used by simply calling `Config. inputOffset` and from other classes/structs by calling `CurrenciesListViewController.Config.inputOffset`